docs/appimage.rst

```diff
@@ -3,14 +3,40 @@ Via AppImage
 
 `AppImage <https://appimage.org/>`_ is a format for distributing portable software on Linux without needing superuser permissions to install the application. It tries also to allow Linux distribution-agnostic binary software deployment for application developers, also called Upstream packaging. 
 
-AppImage allows xonsh to be run on any AppImage supported Linux distributive without installation and root access.
+In short the AppImage is the one executable file which contains the xonsh and Python. AppImage allows xonsh to be run on any AppImage supported Linux distributives without installation and root access.
```

Comment:
`In short the AppImage is one executable file which contains both xonsh and Python. AppImage allows xonsh to be run on any AppImage supported Linux distribution without installation or root access.`

Reply:
Thanks!